Neos.Neos/Migrations/ContentRepository/Version20230809164806.yaml

```
@@ -0,0 +1,15 @@
+comments: 'Migrate shortcut nodes: add httpStatusCode property'
+migration:
+  -
+    filters:
+      -
+        type: 'NodeType'
+        settings:
+          nodeType: 'Neos.Neos:Shortcut'
```

If someone create a own node type with the super type `Neos.Neos:Shortcut`, we still need to add this value

```suggestion
          nodeType: 'Neos.Neos:Shortcut'
          withSubTypes: true
```

---

We will be using a general setting so the Migration is no longer needed